test/OpenTelemetry.Exporter.Prometheus.Tests/PrometheusSerializerTests.cs

```diff
@@ -113,16 +113,25 @@ public void GaugeOneDimension()
                 .AddInMemoryExporter(metrics)
                 .Build();
 
-            var counter = meter.CreateCounter<long>("test_counter");
-            counter.Add(123, new KeyValuePair<string, object>("tagKey", "tagValue"));
+            var tags1 = new List<KeyValuePair<string, object>>();
```

Comment:
Is there a way to simplify this by avoiding allocation and putting the logic directly at Ln124?

Reply:
E.g. https://github.com/open-telemetry/opentelemetry-dotnet/blob/d421cf030678631ef47743ddf4f6c974dcd3e85f/docs/metrics/learning-more-instruments/Program.cs#L64